src/cpp/handler/refreshworker.h

```diff
@@ -50,6 +48,7 @@ class RefreshWorker : public Deferred
 	bool ignoreErrors_;
 	ZrpcManager *proxyControlClient_;
 	ZrpcRequest *req_;
+	Connection finishedConnection;
```

Comment:
Make this `finishedConnection_`, for consistency with other members.

Background: we typically use trailing underscores on private members when d pointers are not used.

Reply:
updated